editor/gui/inspector_dock.h

```diff
@@ -17,6 +17,8 @@ class InspectorDock
 private:
 	static InspectorDock* s_Singleton;
 	constexpr static unsigned int s_InputTextBufferSize = 256;
+	static bool s_IsSorted;
+	static Vector<String> s_Components;
```

Comment:
no need to add these static variables here, declare them in the function itself, m_* notation is also not needed :p. There are many static variable declarations in editor methods, refer them.
Tese are as good as normal members since this is a singleton kinda class.
Also force push to 1 commit so we can merge this

Reply:
Fixed